schemas/sdk.schema

```diff
@@ -999,7 +999,7 @@
                 "unrecognizedPrompt": {
                     "$kind": "Microsoft.IActivityTemplate",
                     "title": "Unrecognized prompt",
-                    "description": "Message to send if user response is not recognized.",
+                    "description": "Message to send when the recognizer does not understand the user input.",
                     "examples": [
```

Comment:
Nice job running the latest bf dialog:merge--this has my changes from today!  (Or did you just edit file directly?)

Reply:
I ran the latest from Master :)